setup.py

```diff
@@ -81,13 +81,23 @@ def get_version(version_file):
     ],
     python_requires=">=3.6",
     # What does your project relate to?
-    keywords="skeleton",
+    keywords="Posture and Exposure Reports",
```

Comment:
This should be a list of lower-case keywords like:
```suggestion
    keywords="posture exposure report",
```

Reply:
Resolved in e294e62